code/modules/cargo/packs.dm

```diff
@@ -1645,6 +1645,44 @@
 			qdel(D)
 			new /mob/living/simple_animal/pet/dog/corgi/Lisa(.)
 
+/datum/supply_pack/critter/corgi/exotic
+	name = "Exotic Corgi Crate"
+	desc = "Corgis fit for a king, these corgis come in a unique color to signify their superiority. Comes with a cute collar!"
+	cost = 5500
+	crate_name = "exotic corgi crate"
+
+/datum/supply_pack/critter/corgi/exotic/generate()
+	. = ..()
+	var/mob/living/simple_animal/pet/dog/corgi/D = locate() in .
+	D.name = "exotic corgi"
+	D.color = rgb(rand(0, 255), rand(0, 255), rand(0, 255))
+
+/mob/living/carbon/human/talking_corgi/handle_status_effects()
+	..()
+	if(prob(5))
+		say(pick("Woof.", "Bark.", "Arf.", "Woof woof.", "Give me a treat."))
+
+/datum/supply_pack/critter/corgi/talking
+	name = "Talking Corgi Crate"
+	desc = "The smart folks at NenoTresen have designed the perfect corgi! It talks! Buy it! Now! Now!" //the spelling mistake is intentional
+	cost = 4500
+	contains = list(/mob/living/carbon/human/talking_corgi)
+	crate_name = "talking corgi crate"
+
+/datum/supply_pack/critter/corgi/talking/generate()
+	. = ..()
+	var/mob/living/carbon/human/talking_corgi/H = locate() in .
+	H.reagents.add_reagent("whiskey", 80)
+	H.put_in_hands(new /obj/item/reagent_containers/food/drinks/bottle/whiskey)
+	H.equip_to_appropriate_slot(new /obj/item/clothing/suit/hooded/ian_costume)
+	H.equip_to_appropriate_slot(new /obj/item/clothing/head/hooded/ian_hood)
+	H.hair_style = "Skinhead"
+	H.facial_hair_style = "Neckbeard"
+	H.gender = MALE
+	H.update_body()
+	H.update_hair()
+	H.name = "talking corgi"
```

Comment:
why not have all this information set when the talking corgi mob initializes, rather than when the crate does?

Reply:
also maybe think about making the corgi costume an actual coded outfit so you dont have to use equip_to_appropriate_slot

Reply:
i didnt care

Reply:
what

Reply:
![](https://i.imgur.com/diZ2Azt.png)